patterns/base/ul/pattern.yml

```diff
@@ -0,0 +1,12 @@
+name: Unordered List
+description: Unordered list of items
+twig: ./ul.twig
+sass: ./ul.scss
+category: base
+data:
+  ul:
+    class: base--ul
+  li:
+    class: base--li
+    text: List item 1
+    class: base--li
```

Comment:
note @scottnath : i am adding `li` values here b/c when i use the `include` in the `ul` or `ol`, it doesn't see `li`s config values.. 
I am not saying it's not possible but need more time to look for solution if you see this as an issue..